packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -572,11 +582,18 @@ export class ExternalRequest {
       if (!linkTable || !linkPrimary) {
         return
       }
+
+      const linkSecondary =
+        linkTable?.primary &&
```

Comment:
I don't think you need the `linkedTable?.primary` check if you have `linkedTable?.primary?.length > 1`

Reply:
You need, if not you have `undefined > 1`, and ts will complain

Reply:
Ah it's TypeScript not JavaScript!